packages/shim-deno/src/deno/stable/functions/consoleSize.ts

```diff
@@ -0,0 +1,6 @@
+///<reference path="../lib.deno.d.ts" />
+
+export const consoleSize: typeof Deno.consoleSize = function consoleSize() {
+  const { columns, rows } = process.stdout;
```

Comment:
Just did some testing and these will be undefined when stdout is piped. I'll update this PR to do what we do with consoleSize in Deno.

Reply:
Actually, I'll merge and open a new PR.

Reply:
Ahh! Good catch. Thanks!!